docs: fix typo (#2020)
wez/wezterm: docs/config/lua/config/ime_preedit_rendering.md

  "Builtin" rendering may truncate displaying of IME preedit
  at the end of window if IME preedit is very long

>   because the rendering does not enable IME preedit to overflow the window

Reviewer: ```suggestion
  because the rendering does not allow the IME preedit to overflow the window
```

  and does not wrap IME preedit to the next line.
  "System" rendering can be useful
  to avoid the truncated displaying of IME preedit

>   but provides worse look and feel than "Builtin" rendering.

Reviewer: ```suggestion
  but has a worse look and feel compared to "Builtin" rendering.
```
